seacrowd/sea_datasets/cc_aligned_sent/cc_aligned_sent.py

```diff
@@ -0,0 +1,167 @@
+_URL = "https://data.statmt.org/cc-aligned/sentence-aligned/"
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+_SUBSETS = ["id_ID", "jv_ID", "su_ID", "th_TH", "vi_VN", "ms_MY", "lo_LA", "km_KH", "my_MM", "cx_PH"]
+
+
+class CCAlignedSentencesDataset(datasets.GeneratorBasedBuilder):
+    """CC Aligned Sentences dataset by Chaudhary et al., (2019)"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "t2t"
+
+    # Add configurations for loading a dataset per language.
+    dataset_names = sorted([f"{_DATASETNAME}_{subset}" for subset in _SUBSETS])
+    BUILDER_CONFIGS = []
+    for name in dataset_names:
+        source_config = SEACrowdConfig(
+            name=f"{name}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=name,
+        )
+        BUILDER_CONFIGS.append(source_config)
+        seacrowd_config = SEACrowdConfig(
+            name=f"{name}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=name,
+        )
+        BUILDER_CONFIGS.append(seacrowd_config)
+
+    # Choose first language as default
+    first_subset = sorted(_SUBSETS)[0]
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_{first_subset}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "Source_Sentence": datasets.Value("string"),
+                    "Target_Sentence": datasets.Value("string"),
+                    "LASER_similarity": datasets.Value("float64"),
+                }
+            )
+
+        if self.config.schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+            features = schemas.text_to_text.features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: DownloadManager) -> List[datasets.SplitGenerator]:
+        """Return SplitGenerators."""
+        # Define some functions for parsing config and URL names
+        def _split_at_n(text: str, n: int) -> Tuple[str, str]:
+            """Split text on the n-th instance"""
+            return ("_".join(text.split("_")[:n]), "_".join(text.split("_")[n:]))
+
+        # Get URL. For cx_PH, the source and target languages are reversed
+        _, subset = _split_at_n(_split_at_n(self.config.name, 5)[0], 3)
+        (source_lang, target_lang) = (subset, "en_XX") if subset == "cx_PH" else ("en_xx", subset)
```

Comment:
tried `en_xx`, didn't work. When it's changed to `en_XX` it works like a charm :)
```suggestion
        (source_lang, target_lang) = (subset, "en_XX") if subset == "cx_PH" else ("en_XX", subset)
```